src/audio_core/stream.h

```diff
@@ -106,6 +111,7 @@ class Stream {
     [[nodiscard]] std::chrono::nanoseconds GetBufferReleaseNS(const Buffer& buffer) const;
 
     u32 sample_rate;                  ///< Sample rate of the stream
+    u64 played_samples;               ///< The current played sample count
```

Comment:
```suggestion
    u64 played_samples{};               ///< The current played sample count
```

Otherwise uninitialized reads can (and do) take place inside of `PlayNextBuffer()`. It introduces a potential uninitialized read vector through the accessor function.